meta-ostro/classes/ostro-image.bbclass

```diff
@@ -230,15 +230,15 @@ ostro_root_authorized_keys () {
 NOISO = "1"
 
 # Add support for compressing images with zip. Works for arbitrary image
-# types. Example: OSTRO_VM_IMAGE_TYPES = "dsk.zip dsk.vdi.zip"
```

Comment:
Does that get rid of the possibility to create a VDI file directly (should one wish to)?